### PR TITLE
Add some things.

### PR DESCRIPTION
**Pull request by klonos**
This is how I think that this document will look better

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,17 @@ this is just a test
 
 ---
 
-## H2 header 1
+## H2 header A
```
Comment (ArmandoDeeb) [inline README.md:6]: I like this change. Thanks 👍

File: README.md [hunk 2]
```diff
@@ -3,13 +3,17 @@ this is just a test
...
+## H2 header B
 
 - this
 - is
 - a
 - list
+
+## H3 header
+
+[this is a link to google calendar](https://google.com/calendar)
```
Comment (ArmandoDeeb) [inline README.md]: I'd rather this link to Google instead. Please fix it.

Comment (klonos) [reply README.md]: Not sure what you mean. Can you elaborate please?

Comment (ArmandoDeeb) [reply README.md]:
```suggestion
[this is a link to google](https://google.com)
```

Comment (klonos) [reply README.md]: Yup, I like it better like so too 👍

**Pull request merged**